fibered_mysql2.gemspec

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "fibered_mysql2/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "fibered_mysql2"
+  spec.version       = FiberedMysql2::VERSION
+  spec.authors       = ["Octothorp"]
+  spec.email         = ["octothorp@invoca.com"]
+
+  spec.summary       = "An adapter for fibered mysql2"
+  spec.homepage      = "https://github.com/Invoca/fibered_mysql2"
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
+  # to allow pushing to a single host or delete this section to allow pushing to any host.
+  spec.metadata["allowed_push_host"] = "https://rubygems.org"
+  spec.metadata["homepage_uri"]      = spec.homepage
```

Comment:
```suggestion
  spec.metadata = {
    "allowed_push_host" => "https://rubygems.org",
    "homepage_uri"      => spec.homepage
  }
```

Reply:
⬆️  suggestion feature is really awesome. First time seeing it